code/lib/cli/src/autoblock/block-storystorev6.ts

```diff
@@ -0,0 +1,40 @@
+import { relative } from 'path';
+import { createBlocker } from './types';
+import { dedent } from 'ts-dedent';
+import type { StorybookConfigRaw } from '@storybook/types';
+
+export const blocker = createBlocker({
+  id: 'storyStoreV7removal',
+  async check({ mainConfig }) {
+    const features = (mainConfig as any as StorybookConfigRaw)?.features;
+    if (features === undefined) {
+      return false;
+    }
+    if (Object.hasOwn(features, 'storyStoreV7')) {
+      return true;
+    }
+    return false;
+  },
+  message(options, data) {
+    const mainConfigPath = relative(process.cwd(), options.mainConfigPath);
+    return `StoryStoreV7 feature must be removed from ${mainConfigPath}`;
+  },
+  log() {
+    return dedent`
+      StoryStoreV7 feature must be removed from your Storybook configuration.
+      This feature was removed in Storybook 8.0.0.
+      Please see the migration guide for more information:
+      https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#story-store-v7
```

Comment:
This points to migration notes for Storybook version upgrade from 6.3.x to 6.4.0. Does it make more sense to point to https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storystorev6-and-storiesof-is-deprecated because user's can also find some information about the `experimental_indexers` API. Though, I think we have to adjust this section a bit to explicitly mention that `features.storyStoreV7` got removed.